modules/process/manager.go

```diff
@@ -48,11 +49,11 @@ type Manager struct {
 
 // GetManager returns a Manager and initializes one as singleton if there's none yet
 func GetManager() *Manager {
-	if manager == nil {
+	managerInit.Do(func() {
```

Comment:
Originally this wasn't a race because the manager was initialised deterministically - has this definitely changed?

Reply:
@zeripath yes I can refactor it throug ...

Reply:
It's fine. Keep the sync.Once it's better that way.